apps/api/src/server/applications/case-officer/__tests__/get-applications.test.js

```diff
@@ -10,21 +10,17 @@ const request = supertest(app);
 const application = applicationFactoryForTests({
 	id: 1,
 	status: 'open',
```

Comment:
As we are no longer using the `open` status in the database, we want to change this probably?